### ISSUE TITLE
put-pipeline hangs forever

### ISSUE THREAD

**Issue by vpicavet** [hobu/greyhound#25]
Steps to reproduce : 
```
vagrant up
vagrant ssh
vagrant@greyhound-dev:/vagrant/examples/data$ ../cpp/put-pipeline  read.xml
```

There it hangs forever. No error message, no information displayed. Same with AZ stadium.

**Comment by connormanning** [comment]
This was due to NPM's default hipache installation having a bad version of dependency `http-proxy` (see https://github.com/hipache/hipache/issues/225), so Greyhound wasn't starting properly.  Fixed in c99bfabced867d892ca490f208d7ff7ae1f9cc76.

**Comment by vpicavet** [comment]
Great, thanks !
